src/main/fc/fc_msp.c

```diff
@@ -2249,6 +2287,10 @@ void mspFcProcessReply(mspPacket_t *reply)
 void mspFcInit(void)
 {
     initActiveBoxIds();
+
+#ifdef USE_RCSPLIT
+    initRCSplitActiveBoxIds();
+#endif
```

Comment:
another solution is needed here, the 64bit for 'boxids' is probably the right choice.